model_rnn/config.py

```diff
@@ -0,0 +1,18 @@
+import os
+import string
+
+
+#File configs.
+TEMP_DIR = 'temp/'
+DATA_FILENAME = 'item_master.csv.gz'
+DATA_FILEPATH = os.path.join(TEMP_DIR, DATA_FILENAME)
+MODEL_FILENAME = 'rnn.model'
+MODEL_FILEPATH = os.path.join(TEMP_DIR, MODEL_FILENAME)
```

Comment:
File paths can be organized like [in here](https://github.com/pomelofashion/ml-data-science/blob/feature/add-content-to-vec/content-to-vec/content_to_vec/config.py).